modules/codegen/src/main/scala/com/twilio/guardrail/generators/ScalaGenerator.scala

```diff
@@ -398,7 +400,7 @@ object ScalaGenerator {
               ..${extraImports}
               import ${buildPkgTerm(List("_root_") ++ pkgName ++ List("Implicits"))}._
               ..${frameworkImplicitName.map(name => q"import ${buildPkgTerm(List("_root_") ++ pkgName)}.${name}._")}
-              import ${buildPkgTerm(List("_root_") ++ dtoComponents)}._
+              ..${dtoComponents.map(x => q"import ${buildPkgTerm(List("_root_") ++ x)}._")}
```

Comment:
as a side note - i think this is the main reason we are having trouble with swagger definitions named `Option` etc ... if we avoided these imports and used fully qualified imports in the generated code, we would be safe - think

Reply:
Yeah; After the qualified DTO PR was merged, I actually don't think this explicit import is even required anymore, other than people using `x-scala-type: List[Foo]`, not having a good way to do effectively `x-scala-type: List[#/definitions/Foo]`